test/v2/fixtures/retrieveBalanceAndTransactions/DocumentDetailsFixture.scala

```diff
@@ -166,7 +166,7 @@ object DocumentDetailsFixture {
        |  "lastClearingReason": "Incoming Payment",
        |  "lastClearedAmount": 3.99,
        |  "statisticalFlag": true,
-       |  "informationCode": "i",
+       |  "informationCode": "K",
```

Comment:
Should "K" be uppercase or lowercase?

Reply:
I have added updated mapping of informationCode now.